helm/harbor-scanner-trivy/values.yaml

```
@@ -74,6 +74,10 @@ scanner:
     ## If the flag is enabled you have to manually download the `trivy.db` file and mount it in the
     ## `/home/scanner/.cache/trivy/db/trivy.db` path (see `cacheDir`).
     skipUpdate: false
+       
+    ## If the flag is enabled you have to manually download the `trivy-java-db` file and mount it in the
+    ## `/home/scanner/.cache/trivy/db/trivy.java-db` path (see `cacheDir`).
```

Is it different from the default path?
```suggestion
    ## `/home/scanner/.cache/trivy/java-db/trivy-java.db` path (see `cacheDir`).
```